src/work-allocation/containers/available-tasks/available-tasks.component.spec.ts

```diff
@@ -23,6 +24,49 @@ class WrapperComponent {
   @ViewChild(AvailableTasksComponent) public appComponentRef: AvailableTasksComponent;
 }
 
+/**
+ * Mock tasks
+ */
+function getTasks(): Task[] {
```

Comment:
This already exists in `src/work-allocation/tests/utils.spec.ts`. You should use that one.

Reply:
Ok, will do